packages/terra-navigation-side-menu/src/_MenuItem.jsx

```diff
@@ -107,6 +108,7 @@ class MenuItem extends React.Component {
           onKeyUp={this.handleKeyUp}
           onBlur={this.handleOnBlur}
         >
+          <VisuallyHiddenText aria-atomic aria-live="assertive" text={isSelected ? 'Selected' : ''} />
```

Comment:
Need translations for this. Looks like form-select has this word translated already: https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/translations/en.json#L8

Reply:
Updated: https://github.com/cerner/terra-framework/pull/761/commits/732e665965ecc0e5d9ed8a01c8c26a6cef892d1a